python/cdp/policies/utils.py

```diff
@@ -0,0 +1,146 @@
+from cdp.openapi_client.models.eth_value_criterion import EthValueCriterion
+from cdp.openapi_client.models.evm_address_criterion import EvmAddressCriterion
+from cdp.openapi_client.models.evm_network_criterion import EvmNetworkCriterion
+from cdp.openapi_client.models.rule import Rule
+from cdp.openapi_client.models.send_evm_transaction_criteria_inner import (
+    SendEvmTransactionCriteriaInner,
+)
+from cdp.openapi_client.models.send_evm_transaction_rule import SendEvmTransactionRule
+from cdp.openapi_client.models.sign_evm_transaction_criteria_inner import (
+    SignEvmTransactionCriteriaInner,
+)
+from cdp.openapi_client.models.sign_evm_transaction_rule import SignEvmTransactionRule
+from cdp.openapi_client.models.sign_sol_transaction_criteria_inner import (
+    SignSolTransactionCriteriaInner,
+)
+from cdp.openapi_client.models.sign_sol_transaction_rule import SignSolTransactionRule
+from cdp.openapi_client.models.sol_address_criterion import SolAddressCriterion
+
+
+def build_policy_rules(initial_rules: list[Rule]) -> list[Rule]:
```

Comment:
Maybe call this function `map_policy_rules_to_openapi_format`, since we will have a counterpart function that maps openapi-formatted response into the custom format